net-print/cups/files/cups-2.3.3-remove-libtool.patch

```diff
@@ -0,0 +1,65 @@
+From 241f812672b82800fe8973825a6f4c372aedb9cd Mon Sep 17 00:00:00 2001
```

Comment:
Link to upstream PR at top always (and gentoo bug)

Reply:
There wasn't a gentoo bug, but I made one. https://bugs.gentoo.org/843638